### DISCUSSION TITLE
Aws s3 ls  SSL validation failed for https://s3.us-east-2.amazonaws.com/ [Errno 2] No such file or directory

### DISCUSSION THREAD

**Discussion by varunsamineni** [aws/aws-cli#7840] [Q&A] [answered]
aws s3 ls

SSL validation failed for https://s3.us-east-2.amazonaws.com/ [Errno 2] No such file or directory

  File "/opt/homebrew/Cellar/awscli/2.11.14/libexec/lib/python3.11/site-packages/awscli/botocore/httpsession.py", line 475, in send
    raise SSLError(endpoint_url=request.url, error=e)
botocore.exceptions.SSLError: SSL validation failed for https://s3.us-east-2.amazonaws.com/ [Errno 2] No such file or directory

**Comment by tim-finnigan** [comment, marked as answer]
SSL validation errors are often caused by certificate issues. Here is more info in the AWS CLI troubleshooting guide: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-troubleshooting.html#tshoot-certificate-verify-failed. 

I also recommend reinstalling the CLI per the documentation on supported installation methods: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
